Modules/_datetimemodule.c

```diff
@@ -663,6 +663,144 @@ set_date_fields(PyDateTime_Date *self, int y, int m, int d)
     SET_DAY(self, d);
 }
 
+/* ---------------------------------------------------------------------------
+ * String parsing utilities and helper functions
+ */
+static inline unsigned int to_int(char ptr)
```

Comment:
The name "ptr" is usually used for pointers.  Please change "ptr" to "ch" here.  On the other hand, I don't think `to_int(ch)` is clearer than `ch - '-'`.  Call it `digit_to_int` or just use the expanded expression.

Reply:
Yeah, this is my mistake. As I mentioned in another comment, I translated this from a macro where it was actually manipulating the pointer directly. When I translated it into a function the variable name didn't get fixed.